networkx/algorithms/approximation/tests/test_maxcut.py

```diff
@@ -0,0 +1,65 @@
+import numpy as np
```

Comment:
numpy is a "soft dependency" (i.e. networkx can be installed and used without it) - the typical way for structuring tests to handle this is to use `pytest.importorskip`. This should resolve the current CI test failure

Reply:
In your test module:
```python
import pytest
np = pytest.importorskip("numpy")
```

instead of ```import numpy```.  This skips the test if the import fails.

Reply:
Thanks for the feedback.
I forgot to remove this explicit numpy dependency. The tests are totally fine without numpy and I will remove it from here.